conda_forge_tick/auto_tick.py

```diff
@@ -688,6 +687,12 @@ def migrator_status(
         for pr_json in attrs.get("PRed", []):
             all_pr_jsons.append(copy.deepcopy(pr_json))
 
+        feedstock_ctx = FeedstockContext(
+            package_name = node,
+            feedstock_name = attrs["feedstock_name"],
```

Comment:
great! You probably want to make sure the attrs has feedstock_name like this and default to node otherwise

`attrs.get("feedstock_name", node)`